spec/factories/users.rb

```diff
@@ -5,6 +5,7 @@
     password { 'passw0rd' }
     sequence(:first_name) { |n| "First_#{n}" }
     last_name { 'Last' }
+    gender { 'non_binary' }
```

Comment:
Oh this is supposed to be `'non-binary'` (dash, not underscore). Feel free to change the Enum definition instead.